src/status_im2/contexts/quo_preview/avatars/group_avatar.cljs

```diff
@@ -3,33 +3,36 @@
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [reagent.core :as reagent]
+            [status-im2.common.resources :as resources]
             [status-im2.contexts.quo-preview.preview :as preview]))
 
 (def descriptor
   [{:label   "Size"
     :key     :size
     :type    :select
-    :options [{:key   :small
+    :options [{:key   :x-small
+               :value "x-small"}
+              {:key   :small
                :value "Small"}
               {:key   :medium
                :value "Medium"}
               {:key   :large
-               :value "Large"}]}
-   {:label "Color"
-    :key :color
-    :type :select
-    :options
-    (map
-     (fn [c]
-       {:key   c
-        :value c})
-     ["#ff0000" "#0000ff"])}]) ; TODO: this is temporary only. Issue: https://github.com/status-im/status-mobile/issues/14566
+               :value "Large"}
+              {:key   :x-large
+               :value "x-Large"}]}
+   {:label "Avatar"
+    :key   :avatar?
+    :type  :boolean}
+   (preview/customization-color-option)]) ; TODO: this is temporary only. Issue: https://github.com/status-im/status-mobile/issues/14566
```

Comment:
Our current guidelines suggest avoiding TODO comments in favor of just having issues. I see the code already had the comment before your PR, but it would be good if you could remove it since you are here. https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#using-todos-comments

Reply:
Hmm, I think we should just try and fix this in this pr? 🤔

Reply:
Yeah, it's unnecessary TODO